analytics/app/src/main/java/com/google/firebase/example/analytics/MainActivity.java

```diff
@@ -19,10 +19,19 @@
 import android.os.Parcelable;
 
 import androidx.appcompat.app.AppCompatActivity;
-
 import com.google.firebase.analytics.FirebaseAnalytics;
 
-public class MainActivity extends AppCompatActivity {
+//  Disregard the below 5 lines which replicate ad_impression data being sent from 3rd parties and are not required for Firebase implementations.
```

Comment:
I believe this is an old comment from when the ad_impression data was being created manually in this file. Can you remove it?